packages/esm-patient-registration-app/src/patient-registration/ui-components/overlay/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
One thing about the file: This overlay is only used inside the `IdField` right now. From my perspective, we could move this file out of the `ui-components` folder and into the folder of the `IdField` to improve discoverability (as long as the overlay isn't going to be used in other locations).